addons/patches/optre/config.cpp

```diff
@@ -1,12 +1,15 @@
 #include "script_component.hpp"
 
+class Mode_FullAuto;
```

Comment:
Needs to be `class LowROF;` instead, then it'll work.

Reply:
...Oh, I didn't read. Derp.